src/constants/networks.ts

```diff
@@ -373,3 +377,5 @@ export const NETWORKS_INFO_MAP: { [id in ChainId]: NetworkInfo } = {
   [ChainId.OASIS]: OasisNetworkInfo,
   [ChainId.OPTIMISM]: OptimismNetworkInfo,
 }
+
+export const NET_WORKS_SUPPORTED: ChainIdType[] = [...SHOW_NETWORKS, ALL_CHAIN_ID].filter((e) => e != ChainId.AURORA)
```

Comment:
I've found we have some ambiguous networks constant here:
 - `ALL_SUPPORT_NETWORKS_ID`
 - `SHOW_NETWORKS`
 - `NET_WORKS_SUPPORTED`

We need some better names here, also add comments to describe which one to use in which situation. Also we need to find and fix what is wrongly used.

If you dont mind, I will fix this and you will do the review on my commit. @nguyenhoaidanh

Reply:
ok. tomorrow I will take a look on it. too many constant, I think I can reduce number of it.

Reply:
updated, please check @namgold